src/software/er_force_simulator_main.cpp

```diff
@@ -30,6 +31,9 @@ int main(int argc, char **argv)
     desc.add_options()("division",
                        boost::program_options::value<std::string>(&args.division),
                        "div_a or div_b");
+    desc.add_options()("realism_config",
+                       boost::program_options::bool_switch(&args.realism),
+                       "realism simulator");  // install terminal flag
```

Comment:
nit: I think to make the arg + variables which are for _enabling/disabling_ (i.e. a boolean value) more descriptive, we should rename them to something like "enable_realism" and for args add a description explaining how setting this arg makes the simulator more realistic (+ same thing for the python arg and variables)